tests/runner.py

```diff
@@ -263,6 +269,7 @@ def create_test_file(name, contents, binary=False):
   'sanity',
   'sockets',
   'interactive',
+  'benchmark'
```

Comment:
nit: comma at the end of each line, makes diffs smaller and edits more uniform

Reply:
Thanks, fixing.